changelogs/CHANGELOG-v0.5.0.md

```diff
@@ -0,0 +1,26 @@
+The v0.5.0 release is a minor release that is the first official release that
+supports the BpfApplication CRD in the bpfman-operator. Another notable
+capability is the ability to manually run the image build/push workflow from
+GitHub actions on an upstream branch. This allows us to manually verify that our
+CI passes with bytecode image changes before merging.
```

Comment:
I wouldn't actually mention this in the release notes since it has zero user facing effect. i.e All infra/CI/automation related changes are essentially invisible in terms of release payload changes.  I would mention that this is the first release where the bpfman-operator is in it's own repository.

Reply:
Done